xbmc/filesystem/VideoDatabaseDirectory.cpp

```diff
@@ -75,6 +75,8 @@ std::string GetChildContentType(const std::unique_ptr<CDirectoryNode>& node)
       return "sets";
     case NODE_TYPE_TAGS:
       return "tags";
+    case NODE_TYPE_VIDEOVERSIONS:
+      return "videoversions";
```

Comment:
There is more to do here to get navigation to work properly.   GetVideoVersions needs adding to `CVideoDatabaseDirectory::GetLabel` which requires another getter in the video db.  You can follow how it works for tags to implement it.

Something like 

```
if (params.GetVideoVersionId() != -1)
   strLabel += videodatabase.GetVideoVersionById(params.GetVideoVersionId());
```